services/docker-compose.yml

```diff
@@ -272,6 +272,22 @@ services:
       - default
       - interactive_services_subnet
 
+  simcore-agent:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/simcore-agent:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}"
+    deploy:
+      mode: global
+      endpoint_mode: dnsrr
+      resources:
+        reservations:
+          cpus: "0.5"
+          memory: "200M"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      LOGLEVEL: ${LOG_LEVEL:-INFO}
```

Comment:
Minor: potentially overwrite with"- SIMCORE_AGENT_S3_PROVIDER=AWS" here?

Reply:
I would leave it as it is, so ti can properly raise an error if not specified. It's simpler to debug.